docs/_includes/footer.html

```diff
@@ -62,6 +62,26 @@
       form.submit();
     };
 
+    function setAnchors() {
```

Comment:
oh! delightful! I've been wishing for this (and avoiding sending links to the GH pages site because it was missing)